### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by ali-7**
fix spelling errors
relate #280

### CODE REVIEW

File: client/src/components/HomePage/modal/LoginModal/index.jsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ class Login extends Component {
                 />
               </div>
               <hr />
-              <h3>Don&apos;t Have an accout ?</h3>
+              <h3>Don&apos;t Have an account ?</h3>
```
Comment (tomduggan85) [inline client/src/components/HomePage/modal/LoginModal/index.jsx]: I think you could remove the space between “account” and the question mark.

File: client/src/components/MainPage/Spectialization/index.jsx [hunk 2]
```diff
@@ -91,10 +92,15 @@ class Specialization extends Component {
...
         <SpecializationCard values={items} />
-        {found && <p className="specialization__showmore">Show All </p>}
+        {found && <p className="specialization__showmore" />}
       </div>
     );
   }
 }
 
+Specialization.propTypes = {
+  history: PropTypes.isRequired,
```
Comment (tomduggan85) [inline client/src/components/MainPage/Spectialization/index.jsx]: You could add a type here, too, for added safety. For example, PropTypes.object.isRequired.

File: server/controller/signup.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ exports.post = async (request, response) => {
               employeeSkillsData = snakeCase(employeeSkillsData);
               await employeeSkills.create(employeeSkillsData);
             });
-            response.status(200).send('Successfule signup, you can login now');
+            response.status(200).send('Successful Signup, You Can Login Now !');
```
Comment (tomduggan85) [inline server/controller/signup.js]: It’s probably not so important for this project, but in general it’s a more flexible design to have your backend send non-user-facing status text, like result: ‘success’ or result: ‘error’ and then have your front end turn that into the user-facing strong “.. you can login now”. That enables you to internationalize / translate your app to other languages much more easily, as that string replacement logic is usually done all in the front end

File: server/database/fakeData.sql [hunk 4]
```diff
@@ -17,15 +17,15 @@ values
...
 ('Nurse (or nursing student)'),
 ('Psychologist'),
 ('Medical Administrator'),
-('Dentist (or dental student'),
-('midwife, or specialist in Family Medicine'),
-('Academic: Porfessor, Lecturer, etc.'),
-('I am a cultural mediator or proffessional traslator'),
-('Medical paraprofessional (or studentexamples include paramedics, physical therapists, nursing, assistants, radiology technicians, etc.');
+('Dentist (or dental student)'),
+('Midwife, or specialist in Family Medicine'),
```
Comment (FarahZaqout) [inline server/database/fakeData.sql]: I think the term is Midwifery.

**Pull request merged**